metricbeat/module/rabbitmq/queue/data.go

```diff
@@ -45,7 +45,7 @@ var (
 		"consumers": s.Object{
 			"count": c.Int("consumers"),
 			"utilisation": s.Object{
-				"pct": c.Int("consumer_utilisation", s.Optional),
+				"pct": c.Int("consumer_utilisation", s.Optional, s.IgnoreAllErrors),
```

Comment:
I guess `s.Optional` is not needed anymore.
```suggestion
				"pct": c.Int("consumer_utilisation", s.IgnoreAllErrors),
```

Reply:
could be ... but imo `Optional` is still a better fit for most scenarios, as it will only let KeyNotFound errors, while `IgnoreAllErrors` ignores any error that happened while converting types.

`Optional` should be used except when it doesn't fit anymore.

Reply:
Yes! It is definitively useful, I meant to remove it from here.

Reply:
oh sure, (got it out of context, my bad)
modifying, thanks!